addons/mail/static/src/models/composer/composer.js

```diff
@@ -312,6 +312,7 @@ function factory(dependencies) {
             thread.refreshFollowers();
             thread.fetchAndUpdateSuggestedRecipients();
             this._reset();
+            return postData;
```

Comment:
It is really strange for this method to return its param instead of its result. Server should be trusted as much as possible, there is no guaranteed how `partner_ids` param was handled.

No matter what we decide here, if signature is changed, docstring should be updated too.